native/src/common/component/View.tsx

```diff
@@ -45,6 +45,9 @@ const ViewBase = styled.View<ViewProps>`
   ${ViewLayout}
 `;
 
+const width_windows = Dimensions.get("window").width;
```

Comment:
En inglés correctamente es:

```suggestion
const windowWidth = Dimensions.get("window").width;
```

Pero podrías crear una constante `Layout.ts` en `src/constant` (que de hecho ya existe), que te de esos valores.